common/commands/src/coconut/generate_freepass.rs

```diff
@@ -142,13 +176,18 @@ pub async fn execute(args: Args, client: SigningClient) -> anyhow::Result<()> {
         None => OffsetDateTime::from_unix_timestamp(args.expiration_timestamp.unwrap())?,
     };
 
-    let now = OffsetDateTime::now_utc();
+    //SAFETY : positive timestamp, so conversion to unsigned will not fail
```

Comment:
but since we're returning an `anyhow::Result<()>` you might as well use `?` operator and remove the unwrap completely

Reply:
Fair point. Fixed in fa1cf7106c7a178fad8d11a99d3ca0a15e09376a